pkgs/data/fonts/inziu-iosevka/default.nix

```diff
@@ -0,0 +1,33 @@
+{stdenv, fetchurl, p7zip}:
+
+stdenv.mkDerivation (rec {
+  version = "1.13.2";
+  name = "inziu-iosevka-${version}";
+
+  package = fetchurl {
+    url = "http://7xpdnl.dl1.z0.glb.clouddn.com/inziu-iosevka-ttfs-${version}.7z";
+    sha256 = "1i7qqcv1x6s1xkp687wq79zqg9ly6a7l5mnmg1iqgfgcbglgjbaw";
+  };
+
+  nativeBuildInputs = [ p7zip ];
+
+  unpackPhase = ''
+    7z x $package
+  '';
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/truetype
+    cp *.ttf $out/share/fonts/truetype
+  '';
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  meta = {
+    description = "Inziu Iosevka font";
+    homepage = https://be5invis.github.io/Iosevka/inziu;
```

Comment:
I'm not sure if it makes sense to package a deprecated font which has been superseded by the Sarasa Gothic one, other than for historical record. Is there any reason anybody would want to use this one instead of Sarasa Gothic?